### PR TITLE
fix(admin-panel/user-updates):make the empty_value None for username

### LINKED ISSUE #469: Unable to persist admin/users/ffadminuser users as they don't have usernames
Admin pages throw an error - if you paste in a username they can then be stored.

### PR DESCRIPTION
**Pull request by gagantrivedi**
update the username field on the admin form to have None as empty value
instead of empty string to play well with the unique constraint on username.

Add test for empty value being None

### CODE REVIEW

File: api/users/forms.py [hunk 1]
```diff
@@ -8,7 +8,11 @@
 
 
 class CustomUserAdminForm(UserChangeForm):
-    username = fields.CharField(required=False, widget=HiddenInput)
+    username = fields.CharField(required=False, widget=HiddenInput, empty_value=None)
+
+    class Meta:
+        model = FFAdminUser
+        fields = UserChangeForm.Meta.fields
```
Comment (matthewelwell) [inline api/users/forms.py:15]: I think this is already the default, right?

**Review by matthewelwell — APPROVED** [review]
